src/routines/routines.ycp

```diff
@@ -912,12 +912,12 @@ define list<string> GetAllInterfaces()
 
 boolean SetLinkUp( string dev_name)
 {
-    return Run( sformat("ip link set %1 up", dev_name) );
+    return Run( sformat("ifup %1", dev_name) );
 }
 
 boolean SetLinkDown( string dev_name)
 {
-    return Run( sformat("ip link set %1 down", dev_name) );
+    return Run( sformat("ifdown %1", dev_name) );
```

Comment:
OK, then we should add this to the commit message:
> Otherwise it could happen that when switching from DHCP -> static, the DHCP address could stay on the device.

But what about the other users of SetLinkUp/Down:

```
src/modules/LanUdevAuto.ycp:218:            SetLinkDown( dev);
src/modules/LanUdevAuto.ycp:238:            SetLinkUp(dev);
# ^ these two are newly added, they should be fine
src/modules/LanItems.ycp:402:            SetLinkDown( dev_name);
src/routines/routines.ycp:936:        ret = SetLinkUp( ifc) && ret;
```

will they work fine with this change?